docs/conf.py

```diff
@@ -28,18 +28,17 @@
 import sys
 from pathlib import Path
 
-sys.path.insert(0, Path("../pecos/").resolve())
-sys.path.append(Path("..").resolve())
+sys.path.insert(0, str(Path("../python").resolve()))
 
-from pecos import __author__, __version__
+from pecos import __version__
 
 # -- Project information -----------------------------------------------------
 
 project = "PECOS"
 copyright = (
-    "2018-2023, The PECOS Developers. Copyright 2018 National Technology & Engineering Solutions of Sandia, LLC (NTESS)"
+    "2018-2023 The PECOS Developers. Copyright 2018 National Technology & Engineering Solutions of Sandia, LLC (NTESS)"
```

Comment:
@qciaran Sphinx requires the comma, see https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-copyright

Reply:
In fact, we can provide a sequence now:
> The value may now be a sequence of copyright statements in the above form, which will be displayed each to their own line.

Reply:
Ah okay, thanks!